packages/examples/packages/wasm/snap.manifest.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/MetaMask/snaps.git"
   },
   "source": {
-    "shasum": "HehP6oFGpT5GZKlzNTVsHCTeTOlBF1044VvfhQGOCEk=",
+    "shasum": "6MsBR6Q/fNzDp5dGRWhBZLIBYu4o31cZRTI9iIUdIn0=",
```

Comment:
Why did this Snap change? 🤔

Reply:
One of the Webpack module IDs changed, I guess because the loader changed slightly.

```
$ diff new.js old.js   
215c215
<           var _build_program_wasm__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(580);
---
>           var _build_program_wasm__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(618);
236c236
<     580: (module, __webpack_exports__, __webpack_require__) => {
---
>     618: (module, __webpack_exports__, __webpack_require__) => {
```